v0/destinations/adj/transform.js

```diff
@@ -44,6 +44,23 @@ function responseBuilderSimple(message, category, destination) {
     response.headers = {
       Accept: "*/*"
     };
+
+    const partnerParamsKeysMap = getHashFromArray(
+      destination.Config.partnerParamsKeys
+    );
+    if (partnerParamsKeysMap) {
+      payload.partner_params = {};
+      Object.keys(partnerParamsKeysMap).forEach(key => {
+        if (message.properties[key]) {
+          payload.partner_params[partnerParamsKeysMap[key]] =
+            message.properties[key];
```

Comment:
```suggestion
            message.properties[key].toString();
```
As adjust is only supporting strings.

Reply:
Done